src/platform/web/ui/session/room/timeline/ReplyPreviewView.js

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2021 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import {renderStaticAvatar} from "../../../avatar";
+import {TemplateView} from "../../../general/TemplateView";
+import {viewClassForEntry} from "../common";
+
+export class ReplyPreviewView extends TemplateView {
+    render(t, vm) {
+        return t.div({ className: "ReplyPreviewView" }, this._renderReplyPreview(t, vm));
+    }
+
+    _renderReplyPreview(t, vm) {
+        // todo: this should probably be called viewClassForTile instead
+        const viewClass = viewClassForEntry(vm);
+        const view = new viewClass(vm, { reply: true, interactive: false });
+        const rendered = view.render(t, vm);
```

Comment:
Calling `render` (or any method really) on a view that is not mounted is a no-no. Also `render` is not part of the `IView` contract but a `TemplateView` impl detail, which `view` may or may not be a instance of. Either way, you can use `t.view` for mounting and rendering a view unconditionally:
```suggestion
        const rendered = t.view(view);
```